tests/e2e/specs/userProfile_spec.js

```diff
@@ -34,4 +35,15 @@ describe('user profile', () => {
     cy.get('[data-test=edit-profile-form-cancel]').click();
     cy.get('[data-test=user-profile-email]').should('contain', newCustomer.email);
   });
+
+  it('changes password', () => {
+    cy.get('[data-test=change-password-button]').click();
+    cy.get('[data-test=change-password-form-currentpassword]').clear().type(oldCustomer.password);
```

Comment:
Is there a reason why you need to call `clear()`? Is there something written before? Because that wouldn't be good. The form should be clean when you enter the tab.

Reply:
I actually copied it from other test and thought maybe I should leave it  just in case 😅

Reply:
Ahh I think it was the editProfile form, in there we have initial values that we need to remove.